lib/resources/port.rb

```diff
@@ -66,7 +66,7 @@ def port_manager_for_os
         LsofPorts.new(inspec)
       elsif os.windows?
         WindowsPorts.new(inspec)
-      elsif ['freebsd'].include?(os[:family])
+      elsif 'freebsd'.include?(os[:family])
```

Comment:
@jquick do we auto-generate predicates for each platform?  If so, we could have `os.freebsd?`.

`os[:family]` is 'bsd' here, which is correct; the platform would be `freebsd`.  I think relying on the fact that one is a substring of the other is not the intent of the code - we are asking specifically if we are on freebsd.

So I agree we need a conditional to match FreeBSD, but I think it should be something higher-level like `os.freebsd?`.